.gitignore

```diff
@@ -41,3 +41,10 @@ crash.log
 
 # Ignore .zip files, such as Lambda Function code slugs.
 **.zip
+
+tests/lenses/rust_wasm32_set_default/Cargo.lock
+tests/lenses/rust_wasm32_set_default/target
+tests/lenses/rust_wasm32_set_default/pkg
+tests/lenses/rust_wasm32_remove/Cargo.lock
+tests/lenses/rust_wasm32_remove/target
+tests/lenses/rust_wasm32_remove/pkg
```

Comment:
nitpick: can do a globbing pattern perhaps under `tests/lenses/*/` to ignore `Cargo.lock`, `target` and `pkg`

Reply:
Another minor nitpick to move these a the top of the file next to other `tests..` rules to ignore that are specific to defradb

Reply:
I do like to be super explicit gitignoring stuff, saves painful annoying issues when stuff disappears due to overly broad pathing. Atm I do prefer the extra lines.

Reply:
Sure, just move these to the top then with the rest of the tests ignores

Reply:
> Sure, just move these to the top then with the rest of the tests ignores

My bad, read that in my email and meant to add a task but forgot. Will move them

- [ ] Move ignores